### ISSUE TITLE
ActivityIndicator on Fabric Not in UIA Tree by Default.

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#11856]
### Problem Description

On Paper, the ActivityIndicator is in the UIA tree, but not focusable by default. On Fabric, the control is not focusable and is not in the UIA tree by default. This behavior should align. 

### Steps To Reproduce

1. Run playground-composition.sln.
2. Add a ActivityIndicator control to a page. 
3. Open AccessibilityInsights to see that the control is not in the UIA tree. 

### Expected Results

Control should be seen in the UIA tree as a progress bar. 

### CLI version

npx react-native -v

### Environment

```markdown
npx react-native info
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by chrisglein** [comment]
Related to #11857.

**Comment by chiaramooney** [comment]
This might have been a xaml convention. Unclear if React Native core expects Activity Indicators to be accessible by default.
